openslides/motions/static/js/motions/site.js

```diff
@@ -1473,16 +1469,7 @@ angular.module('OpenSlidesApp.motions.site', [
         };
         // check if user is allowed to see at least one comment field
         $scope.isAllowedToSeeCommentField = function () {
-            var isAllowed = false;
-            if ($scope.commentsFields.length > 0) {
-                isAllowed = operator.hasPerms('motions.can_see_and_manage_comments') || _.find(
-                        $scope.commentsFields,
-                        function(field) {
-                            return field.public && !field.forState && !field.forRecommendation;
-                        }
-                );
-            }
-            return Boolean(isAllowed);
+            return _.keys($scope.noSpecialCommentsFields).length > 0;
```

Comment:
Hm. This function is declared that is is about user permissions. But you only look into the none special comments fields list regardless of operator.hasPerms('motions.can_see_and_manage_comments') or whatever.

Maybe the name of the function has to be changed if it does something other than before.

Reply:
Renamed this function.